src/common/config.cpp

```diff
@@ -56,4 +115,21 @@ void Config::loadModelParameters(const void* ptr) {
   override(config);
 }
 
+void Config::override(const YAML::Node& params) {
+  for(auto& it : params) {
+    config_[it.first.as<std::string>()] = it.second;
+  }
+}
+
+void Config::log() {
+  YAML::Emitter out;
+  cli::OutputYaml(config_, out);
+  std::string configString = out.c_str();
+
+  std::vector<std::string> results;
+  boost::algorithm::split(results, configString, boost::is_any_of("\n"));
```

Comment:
Please adda  one-line comment what this is (prints the stirng but prepending each line with [config]). Don't make readers reverse-engineer source code if not necessary.

Reply:
Done